pkg/email/help_report_a_problem.go

```diff
@@ -0,0 +1,39 @@
+package email
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+)
+
+// SendReportAProblemEmailInput contains the data submitted by the user to the "report a problem" help form
+type SendReportAProblemEmailInput struct {
+	IsAnonymous            bool
+	ReporterName           string
+	ReporterEmail          string
+	CanBeContacted         bool
+	EasiService            string
+	WhatWereYouDoing       string
+	WhatWentWrong          string
+	HowSevereWasTheProblem string
+}
+
+// SendReportAProblemEmail sends an email to the EASI team containing a user's request for help
+func (c Client) SendReportAProblemEmail(ctx context.Context, input SendReportAProblemEmailInput) error {
+	subject := "Feedback for EASi System"
```

Comment:
According to the [Figma](https://www.figma.com/file/jrc18ENtS3ufj7ttr5vHKW/EASi-Basic-Page-Templates?node-id=28%3A5775), the subject for this should be "EASi Problem Report"